opencti-platform/opencti-front/src/private/components/events/observed_data/Root.jsx

```diff
@@ -13,7 +13,7 @@ import ObservedData from './ObservedData';
 import ObservedDataPopover from './ObservedDataPopover';
 import FileManager from '../../common/files/FileManager';
 import StixCoreObjectHistory from '../../common/stix_core_objects/StixCoreObjectHistory';
-import ContainerHeader from '../../common/containers/ContainerHeader';
+import StixDomainObjectHeader from '../../common/stix_domain_objects/StixDomainObjectHeader';
```

Comment:
Why did you replace `ContainerHeader` by `StixDomainHeader` ?
It adds an action reserved for SDO:
![image](https://github.com/user-attachments/assets/9b5362ef-3df3-4f2c-a67b-8018d4c072bd)

Reply:
Added enrichment to the containerheader and then added the enableEnricher attribute to the containerheader for each of its locations.

Reply:
thanks for that 🙂